conduit/pipeline.go

```diff
@@ -143,7 +147,16 @@ type pipelineImpl struct {
 	importer   *importers.Importer
 	processors []*processors.Processor
 	exporter   *exporters.Exporter
-	round      basics.Round
+
+	blockMetadata         BlockMetaData
+	blockMetadataFilePath string
+}
+
+// BlockMetaData contains the metadata for block file storage
+type BlockMetaData struct {
```

Comment:
nit: could you call this `Metadata` or `PipelineMetadata` to indicate that it has more than just "block" metadata?